pkg/ssh/ssh.go

```diff
@@ -520,6 +520,10 @@ func RunSSHRenderAvalancheNodeConfig(app *application.Avalanche, host *models.Ho
 	if genesisFileExists(host) {
 		avagoConf.GenesisPath = filepath.Join(constants.DockerNodeConfigPath, constants.GenesisFileName)
 	}
+	// make sure that wiz works. TODO switch to RPC over SSH client to access 127.0.0.1 instead
```

Comment:
let's just keep all devnet nodes open without ssh rpc. it is much better for ux experience and not needed to hide behind ssh for devnet

Reply:
if possible remove the TODO

Reply:
agree

Reply:
in the sdk we will have a more liable way once RPC client is ready